bibtex-completion.el

```diff
@@ -1184,93 +1193,81 @@ Return DEFAULT if FIELD is not present in ENTRY."
              (_ value))
          "")))))
 
-(defun bibtex-completion-apa-format-authors (value)
-  "Format author list in VALUE in APA style."
+(defun bibtex-completion-apa-format-authors (value &optional abbrev)
+  "Format author list in VALUE in APA style.
+When ABBREV is non-nil, format in abbreviated APA style instead."
   (cl-loop for a in (s-split " and " value t)
```

Comment:
Uhm, my `lisp-indent-function` seems to have borked your indent.  I don't know much CL, so I'm not sure which one is standard, but I assume yours was.

Reply:
I think mine was non-standard but I used it nonetheless because I think it's more readable. But no big deal.